src/components/Quiz.js

```diff
@@ -31,43 +35,53 @@ const Quiz = () => {
     setScore(totalScore);
     setShouldDisplayScore(true);
   };
+
   const question = data[currentQuestionIndex];
-  return (
+
+  return data.length === 0 ? (
+    <div className="text-center my-5">
+      <Spinner animation="border" variant="primary" size="lg" />
```

Comment:
A spinner is shown in the UI to indicate that something is currently loading. In this case, nothing is loading so it is telling the user wrong information.

We are in the `data.length === 0` condition which is where there are no questions, so we could instead say in the UI that there are no questions

Reply:
"Sorry, there are no questions to display." is displayed now